packages/elements/src/react/common/third-party-providers/social-provider.tsx

```diff
@@ -33,11 +33,13 @@ export function SocialProvider({ asChild, provider, ...rest }: SocialProviderPro
   }
 
   const Comp = asChild ? Slot : 'button';
+  const defaultProps = asChild ? {} : { type: 'button' as const };
```

Comment:
When encapsulated in form elements, buttons without a type default to `submit` causing issues when hitting enter.